src/features/basic/header-hide-controls-button/header-hide-controls-button.ts

```diff
@@ -0,0 +1,38 @@
+import { createFragmentApp } from '@src/utils/vue-fragment-app';
+import TileControlsButton from '@src/components/TileControlsButton.vue';
+import { computedTileState } from '@src/store/user-data-tiles';
+import { getTileState } from './tile-state';
+import { watchEffectWhileNodeAlive } from '@src/utils/watch';
+
+async function onTileReady(tile: PrunTile) {
+  const tileContextControls = await $(tile.frame, C.ContextControls.container);
+  if (!tileContextControls) {
```

Comment:
This check is unnecessary since `await $` will always produce non-nullish values